flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -0,0 +1,582 @@
+                                 **kwargs)
+
+
+def predict(input_data: Union[Channel, List[Channel]],
+            model_info: Union[ModelMeta, Text, int],
+            executor,
+            model_version_info: Optional[Union[ModelVersionMeta, Text]] = None,
+            output_num=1,
+            name: Text = None,
+            **kwargs) -> Union[NoneChannel, Channel, Tuple[Channel]]:
+    """
+    Predictor Operator. Do prediction job with the specific model version.
+
+    :param input_data: List of Channel. It contains the dataset data used in prediction.
+    :param model_info: Information about the model which is in prediction. Its type can be ModelMeta
+                       of py:class:`ai_flow.meta.model_meta.ModelMeta` or Text or int. The model_info
+                       means name in he metadata service when its type is Text and it means id when its type is
+                       int. The ai flow will get the model meta from metadata service by name or id.
+    :param executor: The user defined function in predict operator. User can write their own logic here.
+    :param model_version_info: Information about the model version which is in prediction. Its type can be
+                               ModelVersionMeta of py:class:`ai_flow.meta.model_meta.ModelVersionMeta`
+                               or Text. The model_version_info means version in he metadata service
+                               when its type is Text. The ai flow will get the model meta from metadata
+                               service by version.
+    :param output_num: The output number of the operator. The default value is 1.
+    :param name: Name of the predict operator.
+    :return: Channel or Tuple[Channel].
+    """
+    if isinstance(model_info, ModelMeta):
+        model_meta = model_info
+    elif isinstance(model_info, Text):
+        model_meta = get_ai_flow_client().get_model_by_name(model_info)
+    else:
+        model_meta = get_ai_flow_client().get_model_by_id(model_info)
+
+    if isinstance(model_version_info, ModelVersionMeta):
+        model_version_meta = model_version_info
+    elif isinstance(model_version_info, Text):
+        model_version_meta = get_ai_flow_client() \
+            .get_model_version_by_version(model_version_info,
+                                          get_ai_flow_client().get_model_by_name(model_meta.name).uuid)
+    else:
+        model_version_meta = None
+
+    return user_define_operation(input_data=input_data,
+                                 name=name,
+                                 model_info=model_meta,
+                                 executor=executor,
+                                 model_version_info=model_version_meta,
+                                 output_num=output_num,
+                                 operation_type='predict',
+                                 **kwargs)
+
+
+def evaluate(input_data: Union[Channel, List[Channel]],
```

Comment:
Do we expect the evaluator executor to send events after the evaluation to expose the evaluation result? It might be useful to give a typical behavior of the ops.